static/src/javascripts/projects/commercial/modules/liveblog-adverts.spec.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import { noop } from 'lib/noop';
+// import { noop } from 'lib/noop';
```

Comment:
Would it be better to completely remove this import?

Reply:
yep, should've cleaned that up!